pages/features/list.md

```diff
@@ -186,7 +186,7 @@ Shows which items are the most profitable for the copper cost. (Which ones give
 ### Treecap Cooldown
 
 Dispalys a cooldown bar inspired by the 1.9+ attack indicator. Has Monkey pet support.
-[(Cooldown showcase)](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/69fdead9-a682-4fc9-9830-496443c162e5) 
+[(Cooldown showcase)](/images/Treecap_Cooldown.mp4) 
```

Comment:
Videos apaprently dont get displayed idk

Reply:
Should I remove it then?

Reply:
Up to you, we could just have a link

Reply:
Yay a link! I'll try to do that instead then

Reply:
I meant just leave it rn

Reply:
But I think you're missing an exclamation

Reply:
> But I think you're missing an exclamation

wait that could be it (why it didnt get displayed)

Reply:
> Yay I can finally have a link! I'll try to do that instead then

When you dont have an exclamation or it doesn't point to something displayable, it turns into a hyperlink to the file, which in this case, is a video. I don't mean make the video a link.